src/components/contributions/contributionFilterVM.js

```diff
@@ -0,0 +1,79 @@
+var vm = adminApp.ContributionFilterVM = m.postgrest.filtersVM({
+  full_text_index: '@@',
+  state: 'eq',
+  gateway: 'eq',
+  value: 'between',
+  created_at: 'between'
+});
+
+// Form describer
+vm.formDescriber = [
+  { //full_text_index
+    type: 'main',
+    data: {
+      vm: vm.full_text_index,
+      placeholder: 'Busque por projeto, email, Ids do usuário e do apoio...'
+    }
+  },
+  { //state
+    type: 'dropdown',
+    data:{
+      label: 'Com o estado',
+      name: 'state',
+      vm: vm.state,
+      dataset: [
```

Comment:
I prefer to use ```options``` instead of ```dataset```. Correlates more clearly with the HTML it is going to generate.